docs/docusaurus/docs/application_integration_support.md

```diff
@@ -2,98 +2,69 @@
+| --------------------- | ------------------------- | ----------------------------------------------------------------------------- | --------------------------------- |
+| Data Sources¹         | Snowflake<br/> PostgreSQL | Snowflake<br/>PostgreSQL<br/>Sqlite<br/>Databricks (SQL)<br/>Spark<br/>Pandas | Redshift<br/>MSSQL<br/>MySQL<br/> |
+| Configuration Stores³ | In-app                    | File system                                                                   | None                              |
+| Actions               | Slack                     | Slack ⁴<br/>Email                                                             | None                              |
+| Credential Store      | Environment variables     | Environment variables <br/> YAML⁴                                             | None                              |
+| Orchestrator          | Airflow ⁵                 | Airflow ⁵                                                                     | None                              |
 
-### Integrations
-
-The following table defines the supported GX Cloud and GX OSS integrations.
-
-
-| Integration Type     | GX Cloud    | GX OSS                                                     |
-|----------------------|-------------|------------------------------------------------------------|
-| Data Source          | Snowflake ¹ | Snowflake<br/>Generic SQL ²                                |
-| Configuration Stores | N/A         | Filesystem<br/>AWS S3<br/>GCS Buckets<br/>Azure Blob Store |
-| Notifications        | N/A ³       | Slack ⁴<br/>email<br/>Microsoft Teams<br/>PagerDuty        |
-| Orchestrator         | Airflow ⁵   | Airflow ⁵                                                  |
-
-
-¹ Support for BigQuery will be available in a future release.<br/>
-² Connection strings for various SQL Data Sources are supported, but dialect-specific SQL commands are not.<br/>
-³ Support for Zapier will be available in a future release.<br/>
-⁴ When used with the GX OSS library.<br/>
-⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.
+¹ We've also seen GX work with the following data sources in the past but we can't guarentee ongoing compatibility. These data sources include Clickhouse, Vertica, Dremio, Teradata, Athena, EMR Spark, AWS Glue, Microsoft Fabric, Trino, Pandas on (S3, GCS, Azure), Databricks (Spark), and Spark on (S3, GCS, Azure).<br/>
+² Support for BigQuery in GX Cloud will be available in a future release.<br/>
+³ This includes configuration storage for Expectations, Checkpoints, Validation Definitions, and Validation Result<br/>
+⁴ config_variables.yml<br/>
+⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.<br/>
 
 ### GX components
 
 The following table defines the GX components supported by GX Cloud and GX OSS.
```

Comment:
Throughout the document there are references to GX OSS, but in all of the tables it has been updated to GX Core.  These should be consistent -- "GX OSS" should be replaced with "GX Core" in the text as well as in the table headers.

Reply:
Good catch. Updated all instances of GX OSS with GX Core.